src/script/standard.h

```diff
@@ -80,9 +80,14 @@ struct PKHash : public uint160
     using uint160::uint160;
 };
 
+struct WitnessV0KeyHash;
 struct ScriptHash : public uint160
 {
     ScriptHash() : uint160() {}
+    // These don't do what you'd expect.
+    // Use ScriptHash(GetScriptForDestination(...)) instead.
+    explicit ScriptHash(const WitnessV0KeyHash& hash) = delete;
```

Comment:
What does `explicit` does here?

Reply:
I was going back and forth on how to fix this and left that there :) I think this is a no-op